core/src/main/scala/chisel3/internal/Error.scala

```diff
@@ -101,10 +101,6 @@ private[chisel3] class ErrorLog {
   def warning(m: => String): Unit =
     errors += new Warning(m, getUserLineNumber)
 
-  /** Emit an informational message */
-  def info(m: String): Unit =
-    println(new Info("[%2.3f] %s".format(elapsedTime/1e3, m), None))  // scalastyle:ignore regex
-
```

Comment:
This should only be deprecated and not outright removed. This should be removed in 3.5, but will have been deprecated since 3.2.2.

Reply:
Isn't this an internal object and thus not part of the public API?

Reply:
Ah, good catch. Yes, this should be fine.

Note that I think this will fail backwards compatibility checks (similar to https://github.com/freechipsproject/firrtl/pull/1642) as package private is public Java. That could either be (1) not backported, (2) waived, or (3) straight removed in 3.4.